src/main/java/com/minecolonies/core/generation/defaults/workers/DefaultBakerCraftingProvider.java

```diff
@@ -54,47 +54,47 @@ protected void registerRecipes(@NotNull final Consumer<CustomRecipeBuilder> cons
 
         recipe(BAKER, MODULE_CRAFTING, "bread_dough")
                 .inputs(List.of(new ItemStorage(new ItemStack(Items.WHEAT, 3))))
-                .result(new ItemStack(ModItems.breadDough))
+                .result(ModItems.breadDough.toStack())

```

Comment:
All these replacements will make porting this over a nightmare. I get why, but it's also like 1/100000 on importance score

Reply:
imho this method might exist on 1.20, not sure tho